src/operator/nn/mkldnn/mkldnn_concat.cc

```diff
@@ -20,62 +20,45 @@
 /*!
  * \file mkldnn_concat.cc
  * \brief
```

Comment:
nit: A short brief may be preferred 😄

Reply:
@ciyongch Can you help to add some brief ?

Reply:
seems just file head style, no mkldnn file add brief head